exercises/dominoes/dominoes.exs

```diff
@@ -0,0 +1,12 @@
+defmodule Dominoes do
+
+  @doc """
+  chain?/1 takes a list of integer and returns boolean indicating if chain is possible
+  number to 1 when following the rules:
+    - if number is odd, multiply with 3 and add 1
+    - if number is even, divide by 2
+  """
+  @spec chain?(dominoes :: list) :: boolean
```

Comment:
Since we know the shape of this input pretty exactly, how about defining a type and using that in the typespec?

```elixir
@type domino :: [1..6, 1..6]

@spec chain?([domino] | []) :: boolean
```